docs/guide/upgrade/migrate_v1_v2.md

```diff
@@ -15,17 +15,11 @@ It supports existing AWS resources provisioned by AWSALBIngressController(>=v1.1
 
 1. The AWS LoadBalancer resource created for your Ingress will be preserved.
     
-2. If a numeric TargetPort is used in your service, the AWS TargetGroups created for your Ingress will be re-created.
-
-    !!!warning "downtimes"
-        This would cause downtimes to your service during targets registration into new TargetGroups created.
-    
-    !!!tip "details"
-        * The AWSALBIngressController always used `1` as TargetGroup's port.
-        * The AWSLoadBalancerController will use 
-            * the actual numeric TargetPort as TargetGroup's port if a numeric TargetPort used.
-            * `1` as TargetGroup's port if a lexical TargetPort used.
-        * The AWSLoadBalancerController will automatically create new TargetGroups and cleanup old TargetGroups if any.
+2. The AWSALBIngressController always used `1` as TargetGroup's port. The AWSLoadBalancerController will use 
```

Comment:
the !!!tip annotation is for mkdocs rendering - https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/upgrade/migrate_v1_v2/
Please keep mkdocs the annotation. 

You can also verify the livedocs rendering on your workspace - 
```
pip3 install mike
mike deploy v2.0
mike serve
```
mike is a mkdocs plugin for versioned docs https://github.com/jimporter/mike